cmake/compiler/host-clang/target.cmake

```diff
@@ -0,0 +1,20 @@
+# Configures CMake for using clang
+
+find_program(CMAKE_C_COMPILER   clang   )
+#find_program(CMAKE_CXX_COMPILER clang++ )    # Not used currently
+find_program(CMAKE_OBJCOPY      objcopy )
+#find_program(CMAKE_OBJDUMP     llvm-objdump) # Not used currently
+#find_program(CMAKE_LINKER      ld      )     # Not used currently
+find_program(CMAKE_AR           llvm-ar )
+#find_program(CMAKE_RANLILB     llvm-ranlib ) # Not used currently
+find_program(CMAKE_READELF      readelf )
+find_program(CMAKE_GDB          gdb     )
+
+# Load toolchain_cc-family macros
+# Significant overlap with freestanding gcc compiler so reuse it
+include(${ZEPHYR_BASE}/cmake/compiler/gcc/target_security_fortify.cmake)
```

Comment:
This should be `${ZEPHYR_BASE}/cmake/compiler/${COMPILER}/target_security_fortify.cmake`

Reply:
I just checked, clang doesn't understand fortify at all. Fixed, have a look.